README.md

```diff
@@ -143,8 +164,8 @@ There are two conversions included in the SDK that can be added to your pipeline
 ### Export Functions
 There are two export functions included in the SDK that can be added to your pipeline. 
 	
-- `HTTPPost(string url)` - This function requires an endpoint be passed in order to configure the URL to `POST` data to. Currently, only unauthenticated endpoints are supported. Authenticated endpoints will be supported in the future. 
-- `MQTTPublish()` - Coming Soon
+- `HTTPPost(string url, mimeType string)` - This function requires an endpoint be passed in order to configure the URL to `POST` data to as well as the mime type. Currently, only unauthenticated endpoints are supported. Authenticated endpoints will be supported in the future. If will be `POST`ing JSON or XML you can leverage the `HTTPPostJSON(url string)` or `HTTPPostXML(url string)` respectively as shortcuts so you don't have to specify mimeType yourself. By default, this function will mark the received EdgeX event as pushed in Core Data upon a success response code. 
```

Comment:
Remove words "By default" since there isn't any way, currently, to configure it to not mark as pushed.